mysys/checksum.c

```diff
@@ -34,6 +34,10 @@ ha_checksum my_checksum(ha_checksum crc, const uchar *pos, size_t length)
   extern unsigned int crc32ieee_vpmsum(unsigned int crc, const unsigned char *p,
                                     unsigned long len);
   crc= (ha_checksum) crc32ieee_vpmsum((uint) crc, pos, (uint) length);
+#elif HAVE_ARMV8_CRC
+  extern unsigned int crc32c_aarch64(unsigned int crc, const unsigned char *buffer,
+                                    unsigned long len);
+  crc= (ha_checksum) crc32c_aarch64((uint) crc, pos, (uint) length);
```

Comment:
my_checksum isn't a crc32c algorithm. Its an ieee/iso/ethernet crc32 (1st crc32 polynomial on the CRC wikipedia table)  on a different polynomial (0x1EDC6F41). Run the ```mysql-test/mtr func_math``` test should test this.

Reply:
Ok, I'll remove the crc32c interface from this module
Thanks for your comments.

Reply:
Updated ！ 
:-)